common/lib/xmodule/xmodule/course_module.py

```diff
@@ -380,6 +380,15 @@ class CourseFields(object):
         ),
         scope=Scope.settings
     )
+    enable_ccx = Boolean(
+        display_name=_("Enable CCX"),
+        help=_(
+            "Allow course instructors to assign CCX Coach roles, and allow coaches to manage Custom Courses on edX."
+            " When false, Custom Courses cannot be created, but existing Custom Courses will be preserved."
```

Comment:
You need to add a translators comment to explain what "CCX" and "CCX Coach" means.

http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#python-source-code